libs/partners/mongodb/langchain_mongodb/vectorstores.py

```diff
@@ -489,3 +494,42 @@ async def amax_marginal_relevance_search_by_vector(
             lambda_mult=lambda_mult,
             **kwargs,
         )
+
+    def create_index(
+        self,
+        dimensions: int,
+        filters: Optional[List[Dict[str, str]]] = None,
+        update: bool = False,
+    ) -> None:
+        """Creates a MongoDB Atlas vectorSearch index for the VectorStore
+
+        Note**: This method may fail as it requires a MongoDB Atlas with
+        these pre-requisites:
+            - M10 cluster or higher
+            - https://www.mongodb.com/docs/atlas/atlas-vector-search/create-index/#prerequisites
+
+        Args:
+            dimensions (int): Number of dimensions in embedding
+            filters (Optional[List[Dict[str, str]]], optional): additional filters
+            for index definition.
+                Defaults to None.
+            update (bool, optional): Updates existing vectorSearch index.
+                Defaults to False.
+        """
+        try:
+            self._collection.database.create_collection(self._collection.name)
```

Comment:
1. With the addition of Hybrid Search and VectorStores with full-text search indexes, it makes sense to rename this: `create_vector_search_index` mirroring what is done in `index.py`. 
2. As we've decided to make `_collection` internal, we'll also want to expose methods to manipulate the full-text indexes.

Reply:
Will add this in.